elementor/elementor
# Internal: Remove unused npm script invoking removed grunt task [ED-15601]

## PR Checklist



## PR Type
What kind of change does this PR introduce?

- [x] Other... Please describe: Removing unused `npm script` from `package.json`

## Summary

This PR can be summarized in the following changelog entry:

* Remove unused npm script from package.json.

## Description
An explanation of what is done in this PR

* The grunt qunit task was removed in a6b004af08769b8b5c097516d916fd15ca8dd38f. The npm script invoking it in package.json does not work.

## Test instructions
This PR can be tested by following these steps:

* Run tests in CI. If they still pass, that confirms that the script (which does not work) is not being invoked.
* Perhaps confirm that `npx grunt karma:unit` still works, as that replaced `grunt qunit`.

## Quality assurance
